cypress/support/commands.ts

```diff
@@ -34,4 +34,57 @@
 //       visit(originalFn: CommandOriginalFn, url: string, options: Partial<VisitOptions>): Chainable<Element>
 //     }
 //   }
-// }
\ No newline at end of file
+// }
+Cypress.Commands.add('addUser', function(id, firstName, lastName, userName, eMail, age) { 
```

Comment:
you can improve it by passing object as single parameter. It is just recommendation